src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -239,18 +271,20 @@ void func_80A74398(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-s32 func_80A745E4(EnIk* this, GlobalContext* globalCtx) {
-    if (((this->unk_2FB != 0) || (this->actor.params == 0)) &&
+// Checks to see if action can be interrupted to block attack
+s32 EnIk_ChangeAction(EnIk* this, GlobalContext* globalCtx) {
```

Comment:
The function name is too generic/vague imo,
and the function also checks if the player is attacking,
and there's a bit of randomness with `globalCtx->gameplayFrames & 1`

best I can think of would be `MaybeBlockAttack` but I don't like it